services/api/service.go

```diff
@@ -422,6 +422,7 @@ func (api *RelayAPI) startOptimisticBlockProcessor() {
 	for opts := range api.optimisticBlockC {
 		err := api.simulateBlock(opts)
 		if err != nil {
+			api.log.WithError(err).Error("block simualtion failed")
```

Comment:
"simulation" instead of "simualtion"

Reply:
done! must have been typing too fast hah